CHANGELOG.md

```
@@ -5,7 +5,159 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [0.5.1] - 2026-01-02 "Modular Core"
+## [0.5.2] - 2026-01-03 "Truth & Accuracy"
+
+### Fixed 🐛 CRITICAL
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)